README.md

```diff
@@ -348,6 +349,55 @@ const client = new Client({
 });
 ```
 
+## Logging
+
+By default, all requests made with `bitcoin-core` are logged using [seegno/debugnyan](https://github.com/seegno/debugnyan) with `bitcoin-core:request` as the logging namespace.
+
+Please note that all sensitive data is obfuscated before calling the logging function.
+
+### Example
+
+Example output with `DEBUG=bitcoin-core:request`:
+
+```javascript
+client.getTransactionByHash('b4dd08f32be15d96b7166fd77afd18aece7480f72af6c9c7f9c5cbeb01e686fe');
+
+//{"name":"bitcoin-core","hostname":"<your hostname>","pid":10803,"component":"request","level":20,"request":{"headers":{"host":"127.0.0.1:18333","accept":"application/json"},"id":"bd9a2775-9226-4b95-94c3-201e5d9ba265","method":"GET","type":"request","uri":"http://127.0.0.1:18333/rest/tx/b4dd08f32be15d96b7166fd77afd18aece7480f72af6c9c7f9c5cbeb01e686fe.json"},"msg":"Making request bd9a2775-9226-4b95-94c3-201e5d9ba265 to GET http://127.0.0.1:18333/rest/tx/b4dd08f32be15d96b7166fd77afd18aece7480f72af6c9c7f9c5cbeb01e686fe.json","time":"2017-01-30T11:11:49.690Z","v":0}
+
+//{"name":"bitcoin-core","hostname":"<your hostname>","pid":10803,"component":"request","level":20,"request":{"duration":39,"id":"bd9a2775-9226-4b95-94c3-201e5d9ba265","response":{"body":"b4dd08f32be15d96b7166fd77afd18aece7480f72af6c9c7f9c5cbeb01e686fe not found\r\n","headers":{"content-type":"text/plain","date":"Sat, 28 Jan 2017 12:56:58 GMT","content-length":"76","connection":"close"},"statusCode":404},"type":"response","uri":"http://127.0.0.1:18333/rest/tx/b4dd08f32be15d96b7166fd77afd18aece7480f72af6c9c7f9c5cbeb01e686fe.json"},"msg":"Received response for request bd9a2775-9226-4b95-94c3-201e5d9ba265","time":"2017-01-30T11:11:49.711Z","v":0}
+```
+
+#### Custom logging function
```

Comment:
So instead I'd prefer to change:
- `### Example` to `#### Example`
- `#### Custom logging function` to `### Custom logging function`.
```
## Logging 
#### Example 
### Custom logging function
#### Example
```